static/styles/landing-page.scss

```diff
@@ -3013,6 +3013,11 @@ nav ul li.active::after {
         justify-content: center;
         align-items: center;
     }
+
+    .portico-landing.hello .apps .left-side .platform-icons .group {
+    		margin: 50px 30px;
+    		display: block;
+    }
```

Comment:
I like this fix and merged it, but I think the other bring clearly broken here is that these are supposed to be inline-block (i.e. the icons all on a row together).  There's plenty of space for that, at least...

![image](https://user-images.githubusercontent.com/2746074/39726334-91996440-5203-11e8-9a9e-aa36f2f5a71d.png)

Reply:
I can shrink them on mobile so they all fit on one line—didn't realize the breaking wasn't intentional.